Frameworks/CoreText/CTParagraphStyle.mm

```diff
@@ -106,8 +106,8 @@ bool GetValue(CTParagraphStyleSpecifier specifier, size_t size, void* outValue)
         if (specifier <= kCTParagraphStyleSpecifierCount && outValue != nullptr && size == __CTParagraphStyleSizeTable[specifier]) {
             switch (specifier) {
                 case kCTParagraphStyleSpecifierTabStops:
```

Comment:
okay.  If we have sample code that is using it, we should remove it.

---
In reply to: [88520384](https://github.com/Microsoft/WinObjC/pull/1409#discussion_r88520384) [](ancestors = 88520384)